src/Listener/SaveTrelloIdToDatabase.php

```diff
@@ -117,62 +115,31 @@ private function rememberLastUsedBoardId(string $shortLink): void
 
     private function attachLabelsToCardBasedOnForumTags(Discussion $discussion, Card $card, string $shortLink)
     {
-        if ($client = $this->createTrelloApiClient()) {
+        $client = resolve(Client::class);
```

Comment:
Any reason we are resolving the `$client` at runtime and not injecting it via the contructor?

If the only reason we need `$client` here is to pass it on, then we don't need it at all